{"name": "test_landslide_case_start", "status": "failed", "statusDetails": {"message": "AssertionError: landslide is running a case!\nassert False", "trace": "get_test_id = False\n\n    def test_landslide_case_start(get_test_id):\n>       assert get_test_id, \"landslide is running a case!\"\nE       AssertionError: landslide is running a case!\nE       assert False\n\ntest_001_pdu_establishment.py:48: AssertionError"}, "attachments": [{"name": "log", "source": "c1009fd9-9d52-4165-95cd-a897c5db683f-attachment.txt", "type": "text/plain"}], "start": 1566877010967, "stop": 1566877010967, "uuid": "2e3d8394-94b7-4042-92fa-2d15700beb4a", "historyId": "9eb2b9f80043f88b2f58d008f2727442", "testCaseId": "b3f4749ff5dc127898a5c843642053f0", "fullName": "test_001_pdu_establishment#test_landslide_case_start", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_001_pdu_establishment"}, {"name": "subSuite"}, {"name": "host", "value": "ubuntu18"}, {"name": "thread", "value": "29876-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_001_pdu_establishment"}]}